test/_/no_namespace/BUILD.bazel

```
@@ -0,0 +1,20 @@
+load("@rules_proto//proto:defs.bzl", "proto_library")
+load("//:index.bzl", "ts_proto_library")
+
+
+SRCS = glob(["**/*.proto"])
+
+proto_library(
+    name = "protos",
+    srcs = SRCS,
+)
+
+ts_proto_library(
+    name = "noNamespace",
+    outs = [
+        src.replace(".proto", ".ts")
+        for src in SRCS
+    ],
+    deps = [":protos"],
+    visibility = ["//test/noNamespace:__pkg__"]
```

```suggestion
    visibility = ["//test/no_namespace:__pkg__"]
```
pattern is //{relative/path/to/BUILD}:{name}  so basically we want this target to be only visible to any target within //test/no_namespace directory.